consensus/src/execution_ctx.rs

```diff
@@ -295,6 +295,20 @@ impl<'a, T: Operations + 'static> ExecutionCtx<'a, T> {
         phase: Arc<Mutex<C>>,
         msg: Message,
     ) -> Option<Message> {
+        // If it's a message from a future iteration of the current round, we
+        // generate the committees so that we can pre-verify its validity.
+        // We do it here because we need the IterationCtx
+        if msg.header.round == self.round_update.round
+            && msg.header.iteration > self.iteration
```

Comment:
Please check that msg.header.prevheader is the same of the current round prev hash

Otherwise we can have some non committeemember error

Reply:
We wouldn't get any committeemember member error because the prevblock is checked in is_valid,
but I added the check so we do not generate a useless committee.